### ISSUE TITLE
Ip Bindings not working

### ISSUE THREAD

**Issue by Garrydfrench** [BiglySoftware/BiglyBT#406]
I use PureVPN in conjunction with BiglyBT. Until recently I didn't have a problem. When I lost the PureVPN connection, BiglyBT would simply stop all downloads.
That is not happening anymore. Although I still have the same bindings checked some torrents are still downloading after the PureVPN connection is lost.
Please check the screenshots of my advanced network settings.
(/url)https://postimg.org/gallery/lzumkhac/
Have I done something wrong with my settings that I am unaware of... Or has there been a change in BiglyBT that I am unaware of?

Basically, can anyone help me with this problem please? I want/need all my torrents to stop once the PureVPN connection is lost.
Thank you.
Garry

**Comment by Garrydfrench** [comment]
Is no one else having this problem?
